crates/p2p/src/tests/common.rs

```diff
@@ -0,0 +1,156 @@
+//! Helper functions for the P2P tests.
+
+use std::time::Duration;
+
+use futures::future::join_all;
+use libp2p::{
+    Multiaddr, PeerId, build_multiaddr,
+    identity::{Keypair, ed25519::Keypair as Ed25519Keypair},
+};
+use tokio_util::{sync::CancellationToken, task::TaskTracker};
+
+use crate::{
+    operator_pubkey::P2POperatorPubKey,
+    swarm::{self, P2P, P2PConfig, handle::P2PHandle},
+};
+
+pub(crate) struct Operator {
```

Comment:
BitVM-bridge context that we need to move away from:
```suggestion
pub(crate) struct User {
```